zee/src/settings.rs

```diff
@@ -8,11 +8,21 @@ use std::{
 
 use crate::error::{Context, Result};
 
-#[derive(Clone, Debug, Default, Deserialize, Serialize)]
+#[derive(Clone, Debug, Deserialize, Serialize)]
 pub struct Settings {
+    pub insert_spaces_for_tab: bool,
     pub theme_name: String,
 }
 
+impl Default for Settings {
+    fn default() -> Self {
+        Self {
+            insert_spaces_for_tab: false,
+            theme_name: "zee-gruvbox".into(),
+        }
+    }
+}
+
```

Comment:
FWIW, in #29 `Settings` should not implement `Default` as there'll be a default settings file corresponding to the "default" impl of `Settings` which gets packaged using `include_dir!`. I think we park adding `insert_spaces_for_tab` for now as that'll be configurable per mode once #29 goes in.

Reply:
Okay.  So, shall I completely revert this change?  :wrench:

Reply:
Yes please :pray:  -- you could create a separate PR based on #29 with these changes if you so feel inclined

Reply:
I will try to change the base branch to the one of #29 and directly revert the change thereby.

Reply:
Done. :wrench: :heavy_check_mark: